src/Query/Parser.php

```diff
@@ -1764,29 +1774,71 @@ public function NewObjectExpression(): AST\NewObjectExpression
         return $expression;
     }
 
+    /** @param array<mixed> $args */
+    public function addArgument(array &$args, bool $useNamedArguments): void
+    {
+        $fieldAlias = null;
+
+        if ($useNamedArguments) {
+            $startToken = $this->lexer->lookahead?->position ?? 0;
+
+            $newArg = $this->NewObjectArg($fieldAlias);
+
+            $key = $fieldAlias ?? $newArg->field ?? null;
+
+            if ($key === null) {
+                throw NoMatchingPropertyException::create(trim(substr(($this->query->getDQL() ?? ''), $startToken, ($this->lexer->lookahead->position ?? 0) - $startToken)));
```

Comment:
Let's use linebreaks please.

```suggestion
                throw NoMatchingPropertyException::create(trim(substr(
                    ($this->query->getDQL() ?? ''),
                    $startToken,
                    ($this->lexer->lookahead->position ?? 0) - $startToken,
                )));
```